### ISSUE TITLE
Radio Group Missing Value Prop

### ISSUE THREAD

**Issue by curly33** [JosephusPaye/Keen-UI#332]
When running the [Basic Radio Group component from the examples page](https://github.com/JosephusPaye/Keen-UI/blob/master/docs-src/pages/UiRadioGroup.vue), I'm getting the following error:

```
vue.runtime.esm.js:427 [Vue warn]: Missing required prop: "value"

found in

---> <UiRadio> at C:\code\packages\keen-ui\src\UiRadio.vue
       <UiRadioGroup> at C:\code\packages\keen-ui\src\UiRadioGroup.vue
```

**Comment by curly33** [comment]
To clarify, this occurs when using `import { UiRadioGroup } from 'keen-ui';`.

I recreated `keen-ui/src/UiRadioGroup.vue` in my local app, and it works just fine (I did have to remove the `<styles></styles>`, though since I don't have the proper loaders set up).

**Comment by JosephusPaye** [comment]
Hi @curly33,

Could you post a snippet of the code causing the issue? Or create an online demo on https://codepen.io/?

**Comment by Svobi** [comment]
Same here. 
Webpack configuration importing into component. Both Radio Group and Checkbox Group react like this.
Component configuration via v-model and own keys config via inline Definition of  :keys="config"

**Comment by lucassmacedo** [comment]
i have the same problem

**Comment by lucassmacedo** [comment]
@Svobi @JosephusPaye @curly33 any solution?

**Comment by StanAnsems** [comment]
First of all, great components, but I have the same issue any solution?

**Comment by SLaks** [comment]
@Svobi No; `v-model` (which it [does pass](https://github.com/JosephusPaye/Keen-UI/blob/master/src/UiRadioGroup.vue#L25)) sets `:value` ([docs](https://vuejs.org/v2/guide/components.html#Form-Input-Components-using-Custom-Events))

I can't figure out why this is happening; in the debugger, the compiled template clearly passes the correct value for `value`.

**Comment by SLaks** [comment]
I investigated this further.

It looks like the problem is that dist/keen-ui.js was compiled by an older version of vue.js that encodes `v-model` differently (in `directives:` instead of `model:`).
Therefore, when that compiled template is run by newer Vue.js builds, it doesn't see the `v-model` value.

@JosephusPaye, can you please upgrade to the current Vue.js and re-publish?

**Comment by SLaks** [comment]
As a workaround, simply import `keen-ui/src/index.js` instead of `keen-ui`.
This import also includes CSS, so don't import the CSS file from `dist`.

You'll need to install `node-sass` and `sass-loader` in `devDependencies`, since you'll now be compiling keen-ui in your own WebPack script.

If you put keen-ui in a DLL (which you probably should), you'll need to update that config too.

**Comment by Shashindran** [comment]
I have the same issue. 
As a workaround, I used individual UiRadios and flex it to display inline.

**Comment by JosephusPaye** [comment]
Version `v1.0.1` has been published with support for Vue latest (v2.4.2).

https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.0.1

This should fix the issue. Closing now, can re-open if the issue persists.
